tests/io/feed_parsers/afpnewsmlnew_test.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2024, 2025 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+import os
+import re
+import datetime
+import unittest
+from pytz import utc
+
+from superdesk.etree import etree
+from superdesk.io.feed_parsers.afp_newsml_1_2_new import AFPNewsMLFeedParser
+
+
+class TestCase(unittest.TestCase):
+    def setUp(self):
+        dirname = os.path.dirname(os.path.realpath(__file__))
+        fixture = os.path.join(dirname, "../fixtures", "afp20.xml")
+        provider = {"name": "Test"}
+        with open(fixture, "rb") as f:
+            self.item = AFPNewsMLFeedParser().parse(etree.fromstring(f.read()), provider)
```

Comment:
but maybe if you use mode `r` in open and then `f.read().encode("iso-8885-1")` it might work.
anyhow all the strings here must be readable, here it must be unicode